tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-initial-dialog.html

```diff
@@ -72,7 +72,7 @@ <h2 id='dialog-title'>[[_title]]</h2>
 import keras
 
 keras.backend.set_session(
-    tf_debug.LocalCLIDebugWrapperSession(tf.Session()), "[[_host]]:[[_port]]")
+    tf_debug.TensorBoardDebugWrapperSession(tf.Session()), "[[_host]]:[[_port]]")
```

Comment:
I think the parens need to be adjusted here, right now the "host:port" string is being passed to set_session() but it should actually be the second arg to TensorBoardDebugWrapperSession